src/components/button/button.ts

```diff
@@ -80,6 +80,10 @@ export class MdButton {
   removeKeyboardFocus() {
     this.isKeyboardFocused = false;
   }
+
+  focus() {
+    this.elementRef.nativeElement.focus();
+  }
```

Comment:
Add a `TODO` to add an `e2e` test for this? (simulating focus in a unit test doesn't really work well)